lib/constable_web/templates/layout/app.html.eex

```diff
@@ -18,8 +18,8 @@
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon-180x180.png" />
     <%= if current_user(@conn) do %>
       <script>window.userToken = "<%= @current_user.token %>";</script>
+      <script>window.shuboxKey = "<%= Application.fetch_env!(:constable, :shubox_key) %>";</script>
```

Comment:
This is sort of outside the scope of what this PR is aiming to do, but how do you feel about moving this JS into a JS file and just storing the token in HTML?

So for example, in the HTML:

```html
<head>
  <meta data-role="shubox-key" data-value="<%= Application.fetch_env!(:constable, :shubox_key) %>">
<head>
```

and then loading it in JS using something like this (untested code):

```js
$(document).onLoad(() => {
  const shuboxKey = $("[data-role=shubox-key]").attr("data-value");
  window.shuboxKey = shuboxKey; 
})
```

I think that'd be a better separation of concerns since we could keep javascript in JS files and markup in HTML files. What do you think?

Reply:
Yeah, this is a good point. To be honest, I never really know what's the best way to pass data from Elixir/Phoenix -> JavaScript. I have used data-attributes in the past, but sometimes find them more work than is needed, especially if the value is global. 

Are you okay if we merge this as is? I wouldn't mind creating an issues for Constable where we revisit how we're handling JavaScript. There's quite a bit of it in html templates. Not sure if we can find a better way to do that.

Reply:
Gotcha, that makes sense. I agree that it's not super clear what the best practice is. I made this suggestion after re-reading the [rails guides] (link in coffeescript) on the subject, but even if that's the route we want to go I agree it makes sense to tackle it all at once as part of a concentrated effort. Totally happy to merge this as-is! 👍

[rails guides]: https://guides.rubyonrails.org/working_with_javascript_in_rails.html#unobtrusive-javascript